src/finufft.cpp

```diff
@@ -754,13 +755,12 @@ int FINUFFT_MAKEPLAN(int type, int dim, BIGINT *n_modes, int iflag, int ntrans,
     // idist, ot, onembed, ostride, odist, sign, flags
     {
       std::lock_guard<std::mutex> lock(fftw_lock);
-
       // FFTW_PLAN_TH sets all future fftw_plan calls to use nthr_fft threads.
       // FIXME: Since this might override what the user wants for fftw, we'd like to
       // set it just for our one plan and then revert to the user value.
       // Unfortunately fftw_planner_nthreads wasn't introduced until fftw 3.3.9, and
       // there isn't a convenient mechanism to probe the version
-      FFTW_PLAN_TH(nthr_fft);
+      // there is fftw_version which returns a string, but that's not compile time
```

Comment:
Seems PR  #526 is trying to address this part.

Reply:
I need to revert this.